### ISSUE TITLE
Filtrování podle značky

### ISSUE THREAD

**Issue by VasaMM** [ludekv/a-drake#330]
Bylo by možné do filtru přidat možnost **Musí mít značku**? Vím, jedná se o obyčejný tag _star_, ale věřím, že existuje spousta uživatelů, kterým slovo tag nic neříká, a toto by jim mohlo přijít vhod...

**Comment by gord951** [comment]
To je hrani se slovicky. Ja zese netusim, co je to "znacka". Tag je zabehany a definovany pojem, jeho zmenou nebo nahradou vzniknou jen zmatky. Pokud nekdo netusi, co je tag, ma si precist navody ke GG.

**Comment by VasaMM** [comment]
Asi jsme se nepochopili. _Značka_ je tag star s prázdnou hodnotou. Ale co je hlavní, tak to, že ho automaticky vytvoří a:Drake při kliknutí na ikonku v detailu keše (viz obrázek). Neznalý uživatel tak vůbec o žádném tagu vůbec nic netuší a ani ho nemusí napadnout, že se jedná o nový tag.
![screenshot_2016-03-02-09-52-59](https://cloud.githubusercontent.com/assets/15856069/13455307/ccd84c9e-e05c-11e5-97dc-7cb043bd143a.png)

**Comment by gord951** [comment]
No, ale to te to trochu tluce s pluginem Star od mikroma, ktery vytvari stejnojmenny tag. Uzivatel tak vlastne klidne muze tusit neco o tagu star, ale je zvykly jej prirazovat podle vlastnich pravidel a tady si jej prida aniz by vedel jak to skutecne pracuje a pak bude prekvapen, pro ma keska tag, kdyz ta jeho kriteria pro prirazeni tagu vlastne nesplnuje.

**Comment by VasaMM** [comment]
No v tom případě by vlastně stačilo jen změnit ikonu na hvězdičku a pak i popis. Tím pádem to bude pěkně spolupracovat s GeoGetem (nějak mě ten plugin nenapadl, přestože ho používám).
Ale to nic nemění na tom, že by bylo fajn přihodit tuto možnost do filtru...

**Comment by ludekv** [comment]
Ono je to schválně udělané tak, aby to bylo kompatibilní s pluginem Star. Jen ta ikonka je holt jiná.

**Comment by gord951** [comment]
Az tak mi ta provazanost s pluginem nevadi, ale vadi mi, ze neni zrejma. Tohle budi dojem samostatneho znacky nekde v aD. A nejsem si jisty, ze stejna ikona by to dostatecne objasnila. Co treba ten popisek upravit na "Pridat znacku Star" nebo jeste lepe "Pridat znacku (tag) Star". Pak by to bylo jasne i mne.

**Comment by ludekv** [comment]
Jenže ty jsi technicky zaměřený uživatel Geogetu. Uživatel a:Drake vůbec GG nemusí používat (i když je to škoda), takže nějaký tag Star mu nic neřekne. A i když GG používá, nemusí vědět o doplňku Star.

**Comment by PontiacCZ** [comment]
Já jsem zrovna ten případ č. 2 :-) - Geoget používám a v něm hodně i tagy, ale tento doplněk jsem doposud neznal (jmenuje se správně [Stars](http://geoget.ararat.cz/doku.php/user:skript:stars), jestli se nemýlím).

**Comment by rimuln** [comment]
Souhlas s Ludkem. Technicky znaly, pouzivam oboji a o tagu ani o doplnku nic nevim. K cemu ze to je;)?

**Comment by gord951** [comment]
Rozumim ti, ale takhle je to zase necitelne pro "technicky zdatnejsi uzivatele GG" :-)

To doplneni o "Star" (o "tag" nemluvim) mi nepripada jen pro technicky zdatne. Jen tim davas najevo, ze znacka ma toto jmeno. A ja a mne podobni si to pak celkem logicky dame dohromady s pluginem. Myslim, ze BFU tim zmaten nebude a pro nas je informacni hodnota toho jednoho sluvka (pripadne se zmenenou ikonou) dost vyrazna.

**Comment by PontiacCZ** [comment]
Ale když se tak dívám teď do a:Drake, tak "Přidat značku" s ikonkou praporku bych si nijak nespojil s onou hvězdičkou z Geogetu bez konzultace s dokumentací (tam jsem to nečetl nebo jsem to asi nějak přeskočil). Změň praporek na hvězdičku a bude to jasné. :)

**Comment by ludekv** [comment]
Ikonku vyměním, ale tím končím. Dělat extra volbu ve filtru je poněkud nesystémové, když stačí nastavit filtr podle tagu. Spíš bych to napsal do dokumentace, ale k tomu bych potřeboval bych pomocníka (i víc) na údržbu wiki, kdyby se toho někdo náhodou chtěl ujmout, budu rád.

**Comment by VasaMM** [comment]
OK, byl to jen nápad...

**Comment by gord951** [comment]
Ludku, zkus pozadat ve foru, zda by tam nebyl nekdo, kdo by byl ochotny prilozit ruku k dilu (k wiki), Tady to asi bude cist minimum lidi. Ja bych do toho klidne sel, ale udrzuju toho docela dost a urcite bych tomu nedokazal venovat tolik casu, kolik by si to zaslouzilo.
